README.md

```diff
@@ -60,6 +60,8 @@ __DATABASE_URL="mysql://your_username:your_pwd@127.0.0.1:3306/your_db_name"__
 Run `php bin/console make:migration`  
```

Comment:
You probably should not run make:migration !
Because the migration files are in the git repository so you'll have duplicated requests.
And if you are in the prod environnement, you won't have the maker bundle usually (so the make: commands will be invalid)